cli/cmd/component-apply.go

```diff
@@ -84,7 +84,7 @@ func applyComponents(lokoConfig *config.Config, kubeconfig string, componentName
 			return diags
```

Comment:
Isn't this commit same as 3228f6af891f9ce6e7f13b511a49ffca35040063 ?

Reply:
Yes, thanks!